### PR TITLE
Added MJML App (v 1.4.0)

### PR DESCRIPTION
**Pull request by SeanSith**
#### Adding a new cask
- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

Satisfies #22602. Name chosen to avoid potential collision with MJML engine (available via NPM).

### CODE REVIEW

File: Casks/mjml-app.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+cask 'mjml-app' do
+  version '1.4.0'
+  sha256 '6b657de8f0fb29db2370d555e6cc384bff8eba7cab8422b18a1ce81df2b285b2'
+
+  # github.com was verified as official when first introduced to the cask
+  url 'https://github.com/mjmlio/mjml-app/releases/download/1.4.0/mjml-app-osx.dmg'
```
Comment (fanquake) [inline Casks/mjml-app.rb]: You need to use #{version} in the url stanza

**Comment by fanquake** [comment]
You'll also need to add an appcast.

**Comment by SeanSith** [comment]
@fanquake Hopefully the above changes fulfill the necessary changes you pointed out.

**Comment by adidalal** [comment]
Thank you for the contribution. It needed some fixes, so they were made in a9e02651ffbdf35222b6f815a79575f8742b7452. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Pull request closed without merging**